.travis.yml

```diff
@@ -19,9 +26,11 @@ addons:
 before_script:
   - export DISPLAY=:99.0; sh -e /etc/init.d/xvfb start
 before_install:
-  - "npm install -g npm@^4"
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH=$HOME/.yarn/bin:$PATH
   - "google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost &"
-bundler_args: --retry 0
+install: yarn --frozen-lockfile
```

Comment:
--frozen-lockfile', "don't generate a lockfile and fail if an update is needed 
I don't think we want to fail the build if we need to update ?
Shouldn't we use the lockfile default